api/mapping/views.py

```diff
@@ -641,18 +642,18 @@ def get_context_data(self, **kwargs):
         pk = self.kwargs.get("pk")
         scan_report = ScanReport.objects.get(pk=pk)
 
+        object_list = get_mapping_rules_list(self.get_queryset())        
         filtered_omop_table = self.kwargs.get("omop_table")
-        source_tables = list(set(
-            [
-                x.source_field.scan_report_table.name
-                for x in context['object_list']
-            ]
-        ))
-
+        source_tables =  list(set([
```

Comment:
Could you move the `filtered_omop_table` line down, as it is unrelated to the two lines around it, who _are_ related by `object_list`.

Reply:
done